src/UI/Chat.module.css

```diff
@@ -1,6 +1,6 @@
 .chat {
 	position: fixed;
-	z-index: 2002;
+	z-index: calc(var(--parley-index-prefix, 100) + 2);
```

Comment:
hoe kan deze z-index eerst hoger zijn dan die van `Header.module.css -> .header` en nu lager?
Hoort ie niet in de zelfde volgorde te staan als eerst?

Reply:
header zit in chat. als die lager is dan chat zou die onder chat tevoorschijn komen (dus nooit zichtbaar). Weet alleen bij deze niet zeker of ik dit goed heb kunnen testen omdat er in header niets stond maar kom ik nog op terug...

Reply:
Ik denk dat deze niet nodig is dan (zie https://github.com/parley-messaging/web-library/pull/43#discussion_r1222585747). Als dat zo is mag die helemaal weg.

Reply:
Resolved, alle onnodige z-indexes zijn nu weg